### PR TITLE
Alternative for _.without

### LINKED ISSUE #33: Equivalent of _.without
Hi,

Is there an elegant way to remove an item from an array in an immutable way without using underscore?

I want `_.without([1,2,3],2)` to return `[1,3]`. 

The original array should remain unmodified.

I'd like to avoid using a syntax involving indexes, slice, splice or whatever because I never remember it and using _ is much simpler.

Is there any simple way with ES next?

### PR DESCRIPTION
**Pull request by tracker1**
Add alternative to `_.without` closes #33

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -369,6 +370,56 @@ It's exposed on _ because previously, like `Underscore`, it was only exposed in
...
+  ```js
+  // Lodash
+  var array = [1, 2, 3]
+  console.log(_.filter(array, 2))
+  // output: [1, 3]
+
+  // Native
+  function without(array, match) {
+    var ret = [];
+    for (var i = 0; i < array.length; i)) {
```
Comment (stevemao) [inline README.md]: I don't like for loops. It creates side effects.

File: README.md [hunk 2]
```diff
@@ -369,6 +370,56 @@ It's exposed on _ because previously, like `Underscore`, it was only exposed in
...
+      return value == match;
+    });
+  }
+  var array = [1, 2, 3]
+  console.log(without(array, 2))
+  // output: [1, 3]
+  
+  // ES2015
+  const without = (array, filtered) => array.filter(n => n != filtered);
+  var array = [1, 2, 3]
```
Comment (stevemao) [inline README.md]: why not just 

```js
array.filter(function(value) {
   return value === match;
});
```

No need to have another abstraction layer `without`.

**Comment by stevemao** [comment]
Thanks for the PR. Can you just include 

```js
array.filter(function(value) {
   return value !== match;
});
```

for native?

We don't recommend for loops in the docs. We don't provide ES6 alternatives either (unless we absolutely need to) 

Also note that lodash is using same value zero algorithm. And you could provide more than one values to filter.

**Comment by tracker1** [comment]
Changed... the es6 alternative, was because it was suggested to do a PR in the respective issue thread.

I actually prefer the ES2015 one-off, and would likely just include it at the top if it made sense, in this case, I'd probably `array.filter(v => v  !== match)` as I prefer it to the full function.

File: README.md [hunk 3]
```diff
@@ -369,6 +370,33 @@ It's exposed on _ because previously, like `Underscore`, it was only exposed in
...
 
 
+### _.without
+:heavy_exclamation_mark:`Lodash only`
+Returns an array where matching items are filtered.
+
+  ```js
+  // Lodash
+  var array = [1, 2, 3]
+  console.log(_.filter(array, 2))
```
Comment (stevemao) [inline README.md]: `_. without(array, 2)`

**Pull request merged**